images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -113,6 +116,29 @@ fi
 
 }
 
+function check_if_new_replica() {
+# Description
+# Check for pre-existing server data into PV, if not found, we assume a new server/replica case
+# Always skip if we are performing a new_deployment
+
+echo "== Checking for existing data on server PV =="
+
+if [[ ! -d ${PV_CONTAINER_DATA_DIR} && ${DEPLOYMENT_STATUS} != new_deployment ]]; then
```

Comment:
The logic for check_if_new_replica is to ensure we are not a new_deployment case and that the server data PV is empty, both need evaluate true for us to consider this is legitimately a new_replica.